Sources_v3/Query/Sorting.swift

```diff
@@ -156,3 +156,19 @@ extension UserListSortingKey {
         return nil
     }
 }
+
+// MARK: - Members Sorting
+
+/// `ChannelMemberListSortingKey` describes the keys by which you can get sorted channel members after query.
+public struct ChannelMemberListSortingKey: RawRepresentable, Equatable, SortingKey {
```

Comment:
Other types for sorting keys we manage via enums but I think the `RawRepresentable` fits better: we don't need to provide explicit `encode` implementation and jump around `.custom(String)` case.